docs/calamari/DevelopOnDolphin.md

```diff
@@ -79,16 +81,14 @@ Polkadot.js Apps is another option, especially if you're running local developme
 
 To use the Dolphin faucet, head to our [Discord channel](https://discord.com/channels/795390654628102165/1055864933692219453).
 
-
 To receive tokens from the faucet, you'll need to get the attention of the faucet bot. Start by typing the command for the token you want, such as `/gimme-dol`. Discord will display a small pop-up asking for your wallet address.
 
-
 Type or paste your wallet address into the address field, then press enter to trigger the faucet request.
 
 You can request each of the four supported tokens (`DOL`, `KAR`, `KSM`, `MOVR`) once per UTC day. If you request the same token twice in the same day, the faucet bot will only respond with the status of your first request of the day. You can check the status of your address at https://faucet.dolphin.community/.
 
 ## Development Tools
 
-* SDK: A set of functionality which connects your public address (polkadot-js), your private addresses (signer) and manages the creation of the payloads required for MantaPay extrinsics. In order to use the Calamari SDK please refer to [SDK](SDK.md) section.
-* Signer: The signer is a desktop application which generates zero knowledge proofs and manages the private keys of your UTXOs. The signer is open sourced and you find instruction to set it up locally at [manta-signer](https://github.com/Manta-Network/manta-signer).
+* `SDK`: A set of functionality which connects your public address (polkadot-js), your private addresses (`manta-signer`) and manages the creation of the payloads required for `MantaPay` extrinsics. In order to use the Calamari SDK please refer to [SDK](SDK.md) section.
```

Comment:
i think the new name is `manta.js`, we should call it "manta.js Javascript SDK" or something similar throughout these docs

Reply:
mentioned manta.js explicitly now